.github/ISSUE_TEMPLATE/3-general_question_template.yml

```diff
@@ -0,0 +1,14 @@
+name: 'General Question'
+description: Create a new ticket for a general question
+title: '🐛 [General Question] - <title>'
+labels: ['question']
+
+body:
+  - type: textarea
+    id: summary
+    attributes:
+      label: 'Summary'
+      description: General Question(s) (for Bugs and Feature Requests, please use the appropriate template)
+      placeholder: '...'
+    validations:
+      required: true
```

Comment:
I figure instead of completely pushing people to github discussions or telegram, we would instead continue giving the ability for questions and then sometime in the future, just get rid of it all together (trying to prevent too much change at once)